core/server/data/schema/schema.js

```diff
@@ -300,5 +300,12 @@ module.exports = {
         created_by: {type: 'string', maxlength: 24, nullable: false},
         updated_at: {type: 'dateTime', nullable: true},
         updated_by: {type: 'string', maxlength: 24, nullable: true}
+    },
+    sessions: {
+        id: {type: 'string', maxlength: 32, nullable: false, primary: true},
+        user_id: {type: 'string', maxlength: 24, nullable: false},
+        session_data: {type: 'text', maxlength: 2000, nullable: false},
```

Comment:
What was the reason here to use type "text"?

Reply:
Because the `json` type defaults to `text` when not inside of postgres, so i stuck with that.

Reply:
Can't you use a regular `varchar` field?

Reply:
`text` is outsourced into a different table.